source/includes/api/_user_management.md

```diff
@@ -293,98 +452,182 @@ curl -X GET http://api.resourcewatch.org/auth/user
 }
 ```
 
+Lists user accounts:
+
+- Only users with role ADMIN have permissions to use this endpoint.
+- Email+password based accounts that have not had their email address confirmed will not be listed by this endpoint.
+- By default, only users belonging to the same apps as the requesting user will be shown. See [Filter by app](#filter-by-app) for more options.
+
 
 #### Filter by name
```

Comment:
Is it worth having a separate section for each possible query param filter? Why not defining all the available query param filters in a table, like we did for other API resources? :)

Reply:
apparently I missed this section in my review :/